src/model.d.ts

```diff
@@ -2202,12 +2202,25 @@ export abstract class Model<TModelAttributes extends {} = any, TCreationAttribut
    * affected rows (only supported in postgres and mssql with `options.returning` true.)
    */
   public static update<M extends Model>(
+    this: ModelStatic<M>,
+    values: {
+        [key in keyof Attributes<M>]?: Attributes<M>[key] | Fn | Col | Literal;
+    },
+    options: UpdateOptions<Attributes<M>> & { returning: true }
+  ): Promise<[affectedCount: number, affectedRows?: M[]]>;
```

Comment:
I think as you split the two signatures, the new signature should be:

```suggestion
  ): Promise<[affectedCount: number, affectedRows: M[]]>;
```